bindata/network/ovn-kubernetes/common/002-rbac-node.yaml

```diff
@@ -79,6 +79,7 @@ rules:
   - get
   - list
   - watch
+  - patch
```

Comment:
This should be under `{{- if .IsNetworkTypeMigration }}` condition.
Is patch required for namespaces, nodes and pods?